SPECS/kernel/kernel.spec

```diff
@@ -404,6 +420,8 @@ ln -sf linux-%{uname_r}.cfg /boot/mariner.cfg
 %{_libdir}/perf/examples/bpf/*
 %{_libdir}/perf/include/bpf/*
 %{_includedir}/perf/perf_dlfilter.h
+%{_unitdir}/cpupower.service
+%{_sysconfdir}/sysconfig/cpupower
```

Comment:
this should be `config(noreplace)` shouldn't it?

Reply:
100%. changing